Lib/test/test_zoneinfo/_support.py

```diff
@@ -66,11 +66,38 @@ def setUpClass(cls):
         super().setUpClass()
 
     @contextlib.contextmanager
-    def tzpath_context(self, tzpath, lock=TZPATH_LOCK):
+    def tzpath_context(self, tzpath, block_tzdata=True, lock=TZPATH_LOCK):
+        def pop_tzdata_modules():
+            tzdata_modules = {}
+            for modname in list(sys.modules):
+                if not modname.startswith("tzdata"):
+                    continue
```

Comment:
This can now be removed since `modname.split(".", 1)[0] != "tzdata"` also handles it.